apps/server/src/shared/domain/entity/legacy-board/legacy-boardelement.entity.ts

```diff
@@ -3,33 +3,33 @@ import { EntityId } from '../../types';
 import { BaseEntityWithTimestamps } from '../base.entity';
 import { LessonEntity } from '../lesson.entity';
 import { Task } from '../task.entity';
-import { ColumnBoardTarget } from './column-board-target.entity';
+import { ColumnBoardNode } from '../boardnode';
 
-export type BoardElementReference = Task | LessonEntity | ColumnBoardTarget;
+export type LegacyBoardElementReference = Task | LessonEntity | ColumnBoardNode;
```

Comment:
Get rid of entity column-board-target and use directly columnBoardNode instead.
So we have an actual entity reference, for all legacy-board elements. For column board, is the same as for task and lesson